### DISCUSSION TITLE
Curso FullStack.NET - Paginação

### DISCUSSION THREAD

**Discussion by mkydouglas** [balta-io/forum#586] [Tira Dúvidas] [answered]
Fala, Balta! Tudo bem?

Na aula, o senhor diz para termos um construtor vazio na classe Response para que as conversões aconteçam corretamente. Em seguida, na classe PagedResponse, o senhor não criou um construtor vazio, por quê? Não vai dar problema nas conversões?

Obrigado!!

**Comment by Alekseidasilva** [comment, marked as answer]
Boa Noite @mkydouglas , acredito que nao, pq ja estou mais avancado e nao tive nenhum problema com isso!

**Reply by andrebaltieri** [reply]
Exatamente!! Não tem problema hahahah
